package-lock.json

```diff
@@ -7790,32 +7790,110 @@
       }
     },
     "node_modules/@leafygreen-ui/select": {
-      "version": "14.0.2",
-      "resolved": "https://registry.npmjs.org/@leafygreen-ui/select/-/select-14.0.2.tgz",
-      "integrity": "sha512-9SQFkyyXG0x0p5x64yTacVxVUSaRqepHGmJRJqlr0cAeAxJ1S9CT3Cj+V9rMWTbIUlZ6Tgma4WWGt4cav3LZNQ==",
+      "version": "14.1.8",
+      "resolved": "https://registry.npmjs.org/@leafygreen-ui/select/-/select-14.1.8.tgz",
+      "integrity": "sha512-wE7umGJRF3lIlQjkbg54JwUG/D/YSPwAvNby/JRYIKKPZ9nsldCL5qks2uBF5PW+V1Kn4ikl+Q7kdz3i+uon5g==",
       "license": "Apache-2.0",
       "dependencies": {
-        "@leafygreen-ui/button": "^22.0.2",
-        "@leafygreen-ui/emotion": "^4.0.9",
-        "@leafygreen-ui/form-field": "^2.0.2",
-        "@leafygreen-ui/hooks": "^8.3.4",
-        "@leafygreen-ui/icon": "^13.1.2",
-        "@leafygreen-ui/input-option": "^3.0.2",
-        "@leafygreen-ui/lib": "^14.0.2",
-        "@leafygreen-ui/palette": "^4.1.3",
-        "@leafygreen-ui/popover": "^13.0.2",
-        "@leafygreen-ui/tokens": "^2.11.3",
-        "@leafygreen-ui/typography": "^20.0.2",
-        "@lg-tools/test-harnesses": "^0.1.4",
+        "@leafygreen-ui/button": "^23.1.6",
+        "@leafygreen-ui/emotion": "^4.1.1",
+        "@leafygreen-ui/form-field": "^2.0.12",
+        "@leafygreen-ui/hooks": "^8.4.1",
+        "@leafygreen-ui/icon": "^13.4.0",
+        "@leafygreen-ui/input-option": "^3.0.12",
+        "@leafygreen-ui/lib": "^14.2.0",
+        "@leafygreen-ui/palette": "^4.1.4",
+        "@leafygreen-ui/popover": "^13.0.11",
+        "@leafygreen-ui/tokens": "^2.12.2",
+        "@leafygreen-ui/typography": "^20.1.9",
+        "@lg-tools/test-harnesses": "^0.2.0",
         "@types/react-is": "^18.0.0",
         "lodash": "^4.17.21",
         "polished": "^4.1.3",
         "react-is": "^18.0.1"
       },
       "peerDependencies": {
-        "@leafygreen-ui/leafygreen-provider": "^4.0.2"
+        "@leafygreen-ui/leafygreen-provider": "^4.0.7"
+      }
+    },
+    "node_modules/@leafygreen-ui/select/node_modules/@lg-tools/test-harnesses": {
+      "version": "0.2.0",
+      "resolved": "https://registry.npmjs.org/@lg-tools/test-harnesses/-/test-harnesses-0.2.0.tgz",
+      "integrity": "sha512-vxHT3od7xkfr0Ls7R6ZetSInrpNMag3rxjAgueYkbu0m51WtDNznx0JBWawQ4o+mO6qCraYR5/Zzs2L4DCfSPg==",
+      "license": "Apache-2.0",
+      "dependencies": {
+        "@testing-library/dom": "9.3.1"
+      }
+    },
+    "node_modules/@leafygreen-ui/select/node_modules/@testing-library/dom": {
+      "version": "9.3.1",
```

Comment:
I remember we were running into issues before when multiple testing-library/dom versions were in our dependency tree, not sure if this is affecting us this time, but might be worth either updating our version also or forcing this one to match ours